public/assets/styles/components/CommunityProfiles.scss

```diff
@@ -1,12 +1,15 @@
 .component.CommunityProfiles {
   .page-header {
+    padding-bottom: 0;
     a {
       position: absolute;
-      top: 2em;
-      left: 2em;
+    }
+    .back-link {
+      @include media(medium-small) {
+        display: none;
+      }
```

Comment:
I would adhere to the way that the browser's datasets page is setup. The back link is unclickable in this version.

Reply:
I can't run the data browser right now and the community profiles mock removes the back link on mobile, so I'm not going to fix this for now.
<img width="290" alt="screen shot 2018-07-31 at 4 51 07 pm" src="https://user-images.githubusercontent.com/6565554/43486522-147353a0-94e2-11e8-85be-91f56ae30d63.png">